storage/lib/build.gradle.kts

```diff
@@ -0,0 +1,107 @@
+import com.google.protobuf.gradle.id
+
+/*
+ * SPDX-FileCopyrightText: 2021 The Calyx Institute
+ * SPDX-License-Identifier: Apache-2.0
+ */
+plugins {
+    id("com.google.protobuf")
+    id("org.jetbrains.kotlin.kapt")
+    id("org.jetbrains.dokka")
+    id("com.android.library")
+    kotlin("android")
+}
+
+android {
+    namespace = "org.calyxos.backup.storage"
+    compileSdk = libs.versions.compileSdk.get().toInt()
+
+    defaultConfig {
+        minSdk = libs.versions.minSdk.get().toInt()
+        targetSdk = libs.versions.targetSdk.get().toInt()
```

Comment:
looks like this is deprecated, probably because we don't need to set this for libraries.

There's some other deprecation warnings (e.g. lintOptions -> lint), maybe already address them in an extra commit?

Reply:
Removed and fixed most others.